applications/RANSApplication/custom_elements/convection_diffusion_reaction_cross_wind_stabilized_element.cpp

```diff
@@ -254,6 +245,8 @@ void ConvectionDiffusionReactionCrossWindStabilizedElement<TDim, TNumNodes, TCon
     BoundedMatrix<double, TDim, TDim> contravariant_metric_tensor;
     double variable_value, relaxed_variable_acceleration;
 
+    BoundedVector<double, TNumNodes> velocity_convective_terms;
+
     for (IndexType g = 0; g < num_gauss_points; ++g) {
```

Comment:
this is an absolutely enormous loop. Consider cutting it up to smaller pieces.

Reply:
I would keep it so that the formulation is easily visible.